### PR TITLE
Delete Erntekalender calendar

### PR DESCRIPTION
**Pull request by adi-stefan**

### CODE REVIEW

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5703904_delete_Erntekalender_year.sql [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+UPDATE c_validcombination
+SET harvesting_year_id = yearMap.toYearId,
+    updated            = TO_TIMESTAMP('2023-09-22 22:22:22.222', 'YYYY-MM-DD HH24:MI:SS.US'),
+    updatedby          = 100
+FROM tmp_year_map yearMap
+WHERE harvesting_year_id = yearMap.fromYearId
+;
+
+UPDATE fact_acct
+SET harvesting_year_id = yearMap.toYearId,
```
Comment (cristinamghita) [inline backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5703904_delete_Erntekalender_year.sql:86]: same in metasfresh repo

**Review by cristinamghita — CHANGES_REQUESTED** [review]
see my comments

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5703904_delete_Erntekalender_year.sql (hunk 1 above)
Comment (adi-stefan) [reply backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5703904_delete_Erntekalender_year.sql]: removed from here, pushed repost to private repo

**Review by cristinamghita — APPROVED** [review]
lgtm

**Pull request merged**